src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -0,0 +1,88 @@
+            new MancalaState([
+                [4, 4, 4, 4, 4, 4],
+                [4, 0, 4, 4, 4, 4],
+            ], 0, [0, 0]),
+        ),
+        TutorialStep.fromMove(
+            $localize`Multiple laps sowing`,
+            $localize`However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 2, 4, 0, 0, 0],
+                [0, 0, 0, 5, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`So, after landing in the house with 3 seeds, a second lap has been started`,
+            TutorialStepFailure.FAILED_TRY_AGAIN(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures during distribution (1/2)`,
+            $localize`If, during some distribution, you pass by one of your house that contains 3 seeds, and drop a fourth seed, you capture the house immediately, then continue the distribution!<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 8, 0, 0, 0, 0],
+                [0, 0, 3, 2, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`Congratulations, you captured 4 seeds.`,
+            MancalaTutorial.YOU_DID_NOT_CAPTURE_ANY_SEEDS(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures during distribution (2/2)`,
+            $localize`If, during some distribution, you pass by one house of the opponent that contains 3 seeds, and drop a fourth seed, the opponent capture the house immediately (while you continue to distribute)!<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 3, 0, 0, 8, 0],
+                [3, 0, 1, 0, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(0))],
+            $localize`There it is, the opponent captured 4 seeds.`,
+            MancalaTutorial.YOU_DID_NOT_CAPTURE_ANY_SEEDS(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures`,
+            $localize`Though, if your very last seed is drop in a house of the opponent that contains 3 seeds, you capture it immediately!<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 1, 1, 3, 8, 0],
+                [2, 7, 2, 0, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(0))],
+            $localize`Congratulations, you captured 4 seeds.`,
+            MancalaTutorial.YOU_DID_NOT_CAPTURE_ANY_SEEDS(),
+        ),
+        TutorialStep.fromMove(
+            TutorialStepFailure.END_OF_THE_GAME(),
+            $localize`At any end of turn, if the number of seed reach 8 or less, the first player capture capture the remaining seeds.<br/><br/>You're playing Dark, end the game by capturing!`,
```

Comment:
At the end of a turn
number of seeds
reaches
captures (et enlever le second "capture")